### ISSUE TITLE
Craft 2 map field migration unknown property: ether\simplemap\records\Map::elementId

### ISSUE THREAD

**Issue by jamesmacwhite** [ethercreative/simplemap#133]
### Description

After updating to the latest version of the Maps plugin. The Craft 2 migration code fails with the following error:

```
Unknown Property: Setting unknown property: ether\simplemap\records\Map::elementId

Migration: ether\simplemap\migrations\m190226_143809_craft3_upgrade

Output:

> delete from {{%plugins}} ... done (time: 0.002s)
> Start map data upgrade
> Upgrade map value Nottingham College Adams Building, Nottingham, UK
Exception: Setting unknown property: ether\simplemap\records\Map::elementId (/app/vendor/yiisoft/yii2/base/Component.php:209)
#0 /app/vendor/yiisoft/yii2/db/BaseActiveRecord.php(324): yii\base\Component->__set('elementId', 191847)
#1 /app/vendor/ether/simplemap/src/migrations/m190226_143809_craft3_upgrade.php(106): yii\db\BaseActiveRecord->__set('elementId', 191847)
#2 /app/vendor/ether/simplemap/src/migrations/m190226_143809_craft3_upgrade.php(48): ether\simplemap\migrations\m190226_143809_craft3_upgrade->_upgrade2()
#3 /app/vendor/craftcms/cms/src/db/Migration.php(56): ether\simplemap\migrations\m190226_143809_craft3_upgrade->safeUp()
#4 /app/vendor/craftcms/cms/src/db/MigrationManager.php(243): craft\db\Migration->up(true)
#5 /app/vendor/craftcms/cms/src/db/MigrationManager.php(163): craft\db\MigrationManager->migrateUp(Object(ether\simplemap\migrations\m190226_143809_craft3_upgrade))
#6 /app/vendor/craftcms/cms/src/services/Updates.php(225): craft\db\MigrationManager->up()
#7 /app/vendor/craftcms/cms/src/controllers/BaseUpdaterController.php(521): craft\services\Updates->runMigrations(Array)
#8 /app/vendor/craftcms/cms/src/controllers/UpdaterController.php(209): craft\controllers\BaseUpdaterController->runMigrations(Array, 'restore-db')
#9 [internal function]: craft\controllers\UpdaterController->actionMigrate()
#10 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#11 /app/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#12 /app/vendor/craftcms/cms/src/web/Controller.php(109): yii\base\Controller->runAction('migrate', Array)
#13 /app/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('migrate', Array)
#14 /app/vendor/craftcms/cms/src/web/Application.php(297): yii\base\Module->runAction('updater/migrate', Array)
#15 /app/vendor/craftcms/cms/src/web/Application.php(683): craft\web\Application->runAction('updater/migrate')
#16 /app/vendor/craftcms/cms/src/web/Application.php(243): craft\web\Application->_processUpdateLogic(Object(craft\web\Request))
#17 /app/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#18 /app/web/index.php(21): yii\base\Application->run()
#19 {main}
```

### Steps to reproduce

1. Have an existing SimpleMap field from Craft 2 placed on an entry
2. Upgrade to v3.4.3
3. Migration runs to update the existing Craft 2 SimpleMap field data and fails with the above.

### Additional info

- Craft version: Craft Pro 3.1.19
- SimpleMap version: 3.4.3
- PHP version: 7.1

**Comment by bfopma** [comment]
I can confirm this issue -- following.

**Comment by Tam** [comment]
@jamesmacwhite @bfopma Would either of you be able to share the contents of your `simplemaps` or `simplemap_maps` tables?

**Comment by jamesmacwhite** [comment]
@Tam Happy to, where would you like me to send it?

**Comment by Tam** [comment]
@jamesmacwhite dev@ethercreative.co.uk

**Comment by jamesmacwhite** [comment]
@Tam Thanks DB export of the simplemap tables has been sent over.

**Comment by Tam** [comment]
This should be fixed as of 3.4.4, but I'd recommend updating to the latest release. Let me know if it fixes your issue.

**Comment by jamesmacwhite** [comment]
@Tam I just updated to the latest release and the migrations have ran successfully.

```
  | 454 | 94 | plugin | m190226_143809_craft3_upgrade | 2019-03-25 13:53:33 | 2019-03-25 13:53:33 | 2019-03-25 13:53:33 | b8bcb5c1-53c5-4e0a-b966-d1e9c0bf8e3d
-- | -- | -- | -- | -- | -- | -- | -- | --
  | 455 | 94 | plugin | m190325_130533_repair_map_elements | 2019-03-25 13:53:33 | 2019-03-25 13:53:33 | 2019-03-25 13:53:33 | 795b2d6e-c395-47cf-8d85-11619c41ce72
```

Many thanks for fixing!
